src/submission/validation-clinical/specimen.ts

```diff
@@ -28,6 +28,13 @@ export const validate = async (
 
   const errors: SubmissionValidationError[] = []; // all errors for record
 
+  const specimen = getSpecimenFromDonor(existentDonor, specimenRecord, errors);
+  if (!specimen) {
```

Comment:
@jaserud  is this still possible ? I believe the merge donor already does the check of registration ? or is that only for donor Ids

Reply:
yea donor ids are the special case, the merge only merges specimen if it exists https://github.com/icgc-argo/argo-clinical/blob/2953152e694a7797383e994dbfa383ea63d2a41d/src/submission/submission-to-clinical/merge-submission.ts#L130

Reply:
Thanks